socialite/src/components/Recommend/Recommend.css

```diff
@@ -0,0 +1,49 @@
+.authenticate-nav{
+	margin: 2rem;
+}
+
+.a-nav-right{
+	text-align: right;
+}
+
+.rounded{
+	padding: 0.4rem 1.5rem;
+	border-radius: 25px !important;
+	background-color: rgba(0,204,187, 0.5);
+	border-style: none;
+}
+
+.rounded:hover{
+	background-color: rgba(0,204,187, 0.9);
+}
+
```

Comment:
this is already in the Dashboard.css file, right?

Reply:
Yes

Reply:
Lemme fix that real quick.

Reply:
Fixed it.